Dockerfile

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM golang:1.14 as api-builder
+
+WORKDIR /go/src/app
+
+COPY ./api .
+
+RUN mkdir -p /go/output/conf \
+    && cp ./conf/*.json /go/output/conf
+
+RUN wget https://github.com/api7/dag-to-lua/archive/v1.1.tar.gz \
+    && tar -zxvf v1.1.tar.gz \
+    && mkdir -p /go/output/dag-to-lua \
+    && mv -u ./dag-to-lua-1.1/lib/* /go/output/dag-to-lua/
+
+RUN go build -o /go/output/manager-api .
```

Comment:
need set  `GOPROXY` before build

```sh
go env -w GOPROXY=https://goproxy.io,direct 
```

It couldn't build in my env before I set GOPROXY.

Reply:
Still not working for me

Reply:
https://github.com/apache/apisix-dashboard/pull/657/commits/331a7e428be596db2e79c1cc3cc91047f6a6a8bd

Reply:
> ```shell
> RUN go env -w GOPROXY=https://goproxy.io,direct 
> ```

this step should be optional. Is it only useful for Chinese developers?

Reply:
Not sure, cc @nic-chen

Reply:
> this step should be optional. Is it only useful for Chinese developers?
Yes, we can add some description for this line.